ecommerce_api_client/auth.py

```diff
@@ -1,24 +1,40 @@
+# pylint: disable=duplicate-code
```

Comment:
@clintonb It's not after @jimabramson suggested I changed the variable name of the block in question. I'll remove it now.